content/docs/2.4/scalers/rabbitmq-queue.md

```diff
@@ -37,6 +37,8 @@ triggers:
 - `protocol`: Protocol to be used for communication. Either `auto`, `http`, or `amqp`. It should correspond with the `host` value. Optional, will autodetect based on the `host` URL if possible.
 - `vhostName`: Vhost to use for the connection, overrides any vhost set in the connection string from `host`/`hostFromEnv`.
 - `queueLength`: DEPRECATED! Use `mode: QueueLength` and `value: ##` instead. Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual number of messages in the queue is 30, the scaler scales to 3 pods. Default is 20 unless `publishRate` is specified, in which case `queueLength` is disabled for this trigger.
+- `useRegex`: In case of `http` protocol and `QueueLength` mode, this parameter allows to use regex to select queue instead full name. Optional.
+- `operation`: Operation that will be applied to compute the number of messages in case of `useRegex` enabled. Either `sum`,`max`, or `avg`. The default value is `sum`.
```

Comment:
```suggestion
- `operation`: Operation that will be applied to compute the number of messages in case of `useRegex` enabled. Either `sum` (default),`max`, or `avg`.
```